.github/workflows/merge.yaml

```diff
@@ -0,0 +1,17 @@
+name: Brigade v2 Merge
```

Comment:
I'm wondering if we should add a comment to these new GitHub Action yaml files mentioning they are currently used for building Windows images and not yet intended for full CI (where it's the brigade.js that does the heavy lifting).  WDYT?

Reply:
Also maybe touching on some of the points mentioned in the PR description, namely:

>The Windows container build also take a very long time. And since most PRs won't touch it, I propose we not configure the v2 branch protections in GitHub to require that particular check pass, and that we not allow pending or in-progress Windows log agent builds to block PRs that don't touch the logger.


Could be good to preserve those in source...

Reply:
Agreed. I will add a commit that adds a little bit more doc/context around why/how we're using GH Actions.